File: sample/string_with_format_sample.m
@implementation StringWithFormatSample

- (NSString *)stringWithFormatTest {
    [NSString stringWithFormat:@"int型：%d", num];
    [NSString stringWithFormat:@"%02i:%02i:%02i", hour, min, sec];
    [NSString stringWithFormat:@"double型：%f", x];
    [NSString stringWithFormat:@"double型：%.1f", f];
    [NSString stringWithFormat:@"%@ %@", rawResult[@"FirstName"], rawResult[@"LastName"]];
    [NSString stringWithFormat:@"NSArray型：%@", ary];
    [[NSString alloc]initWithFormat:@"文字列：%ld", n];
}

@end

File: src/test/resources/string_with_format_sample.swift
class StringWithFormatSample : NSObject, Protocol1, Protocol2 {

    func stringWithFormatTest() -> NSString {
        "int型：\(num)"
        "int型：\(num1 + num2)"
        String(format: "%02i:%02i:%02i", hour, min, sec)
        "double型：\(x)"
        String(format: "double型：%.1f", f)
        "\(rawResult["FirstName"]) \(rawResult["LastName"])"
        "NSArray型：\(ary)"
        NSString.initWithFormat("文字列：%ld"n)

    }

}
